src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/_command_type.py

```diff
@@ -43,8 +43,6 @@ def keyvault_exception_handler(ex):
                            'The vault may not exist or you may need to flush your DNS cache '
                            'and try again later.')
         raise CLIError(ex)
-    else:
-        raise ex
```

Comment:
What fails with this?

Reply:
See below. I can still throw non CLIError, but need to confirmation/encourage from your guys to proceed

```Python
    @ResourceGroupPreparer(name_prefix='cli_test_kv_cert_issuer')
    def test_keyvault_certificate_issuers(self, resource_group):
       .....
        # test admin commands
        self.cmd('keyvault certificate issuer admin add --vault-name {kv} --issuer-name issuer1 --email test@test.com --first-name Test --last-name Admin --phone 123-456-7890', checks=[
            ...
        ])
        # ---> fail below which i guess caused by adding dupe issuers<-----
        self.cmd('keyvault certificate issuer admin add --vault-name {kv} --issuer-name issuer1 --email test@test.com')  
```
```
  File "d:\sdk\azure-cli\src\command_modules\azure-cli-keyvault\azure\cli\command_modules\keyvault\_command_type.py", line 118, in keyvault_command_handler
    return keyvault_exception_handler(ex)
  File "d:\sdk\azure-cli\src\command_modules\azure-cli-keyvault\azure\cli\command_modules\keyvault\_command_type.py", line 47, in keyvault_exception_handler
    raise ex
  File "d:\sdk\azure-cli\src\command_modules\azure-cli-keyvault\azure\cli\command_modules\keyvault\_command_type.py", line 98, in keyvault_command_handler
    result = op(**command_args)
  File "d:\sdk\azure-cli\src\command_modules\azure-cli-keyvault\azure\cli\command_modules\keyvault\custom.py", line 800, in add_certificate_issuer_admin
    raise CLIError("admin '{}' already exists".format(email))
knack.util.CLIError: admin 'test@test.com' already exists
```